app/models/meeting_date.rb

```diff
@@ -2,5 +2,9 @@ class MeetingDate  < ActiveRecord::Base
   has_many :course_meetings
   has_many :courses, through: :course_meetings
 
+  def month_weekday
+    first.strftime('%A %B')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.